wazimap_np/profiles.py

```diff
@@ -1,60 +1,61 @@
 from wazimap.data.utils import get_session, merge_dicts, group_remainder
 from wazimap.geo import geo_data
 
-
 from wazimap_np import (
-    demographics,
+    demographics, agriculture
 )
 
 # ensure tables are loaded
 
-# TODO: keeping this tuple because it is referenced in function code, may be needed to bring back
-# PROFILE_SECTIONS = (
-#     'agriculture',
-#     'business',
-#     'demographics',
-#     'development',
-#     'disasters',
-#     'education',
-#     'elections',
-#     'forests',
-#     'health',
-#     'households'
-# )
-
 PROFILE_SECTIONS = (
     'demographics',
+    'agriculture'
 )
 
+park_geo_codes = {'6099', '14099', '15099', '28088', '28099', '31088', '31099', '33099', '34099', '35099', '43099',
+                  '45099', '49099', '58099', '67099', '68099', '69099', '70099', '72099', '76099', '77099'}
+
+
+def should_have_data(geo_code, geo_level):
+    return geo_level != 'local' and geo_code not in park_geo_codes
```

Comment:
The sample `should_have_data` method I provided in my earlier comment was incorrect. It evaluates to `False` for all local areas because `geo_level != 'local'` always evaluates to `False` for local areas. Does or work instead of and? As in:

```python
def should_have_data(geo_code, geo_level):
   return geo_level != 'local' or geo_code not in park_geo_codes
```

Reply:
I thought of something different but looks like what you proposed and what I've done is essentially the same thing. 

My approach:
```python
return not (geo_level == 'local' and geo_code in park_geo_codes)
```

By DeMorgan's law, when you apply `not` to an expression, `and` becomes `or`.
So,
```python
return geo_level != 'local' or geo_code not in park_geo_codes
```
This is what you proposed.